review-tools/gitaddrev

```diff
@@ -150,6 +150,11 @@ foreach (@ARGV) {
         $WHAT = 'perftools';
         $min_authors = 2;
         $min_otc = 0;
+    } elsif (/--installer$/) {
+        $WHAT = 'installer';
+        $min_authors = 1;
```

Comment:
I might have expected this to match the main repo

Reply:
fixed this up